### PR TITLE
Updated the doc for KubeTask function

### PR DESCRIPTION
**Pull request by shlokchaudhari9**
## Change Overview

In the documentation, KubeTask function had `namespace` as a required field, but at the back-end it's an optional argument. Updated the documentation to be in-sync with the function code-base.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [x] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #XXX

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: docs/functions.rst [hunk 1]
```diff
@@ -118,7 +118,7 @@ This allows you to run a new Pod from a Blueprint.
    :align: left
    :widths: 5,5,5,15
 
-   `namespace`, Yes, `string`, namespace in which to execute
+   `namespace`, No, `string`, namespace in which to execute (if not mentioned will create pod under ``kanister`` namespace)
```
Comment (viveksinghggits) [inline docs/functions.rst:121]: Let's also mention where the pod will be created if we don't specify the namespace.

**Review by viveksinghggits — CHANGES_REQUESTED** [review]

File: docs/functions.rst (hunk 1 above)
Comment (shlokchaudhari9) [reply docs/functions.rst:121]: Wrote `(if not mentioned will create pod under ``kanister`` namespace)` in the arg description. Do we need to add more context or update this line?

Comment (viveksinghggits) [reply docs/functions.rst:121]: aah, I missed that I think. Let's use controller namespace instead of kanister namespace.

Comment (shlokchaudhari9) [reply docs/functions.rst:121]: Cool. I'll make this update. Thanks. 👍🏼

File: docs/functions.rst [hunk 2]
```diff
@@ -118,7 +118,7 @@ This allows you to run a new Pod from a Blueprint.
    :align: left
    :widths: 5,5,5,15
 
-   `namespace`, Yes, `string`, namespace in which to execute
+   `namespace`, No, `string`, namespace in which to execute (if not mentioned will create pod under controller namespace)
```
Comment (viveksinghggits) [inline docs/functions.rst:121]:
```suggestion
   `namespace`, No, `string`, namespace in which to execute (if not mentioned the pod will be created in Controller's namespace)
```

**Review by viveksinghggits — CHANGES_REQUESTED** [review]

File: docs/functions.rst (hunk 2 above)
Comment (shlokchaudhari9) [reply docs/functions.rst]: Done

**Review by viveksinghggits — APPROVED** [review]

File: docs/functions.rst [hunk 3]
```diff
@@ -118,7 +118,7 @@ This allows you to run a new Pod from a Blueprint.
    :align: left
    :widths: 5,5,5,15
 
-   `namespace`, Yes, `string`, namespace in which to execute
+   `namespace`, No, `string`, namespace in which to execute (if not mentioned the pod will be created in controller's namespace)
```
Comment (pavannd1) [inline docs/functions.rst:121]:
```suggestion
   `namespace`, No, `string`, namespace in which to execute (if not specified, the pod will be created in controller's namespace)
```

File: docs/functions.rst [hunk 4]
```diff
@@ -118,7 +118,7 @@ This allows you to run a new Pod from a Blueprint.
    :align: left
    :widths: 5,5,5,15
 
-   `namespace`, Yes, `string`, namespace in which to execute
+   `namespace`, No, `string`, namespace in which to execute (if not specified, the pod will be created in controller's namespace)
```
Comment (pavannd1) [inline docs/functions.rst:121]:
```suggestion
   `namespace`, No, `string`, namespace in which to execute (the pod will be created in controller's namespace if not specified)
```

**Pull request merged**